### PR TITLE
Add load tables to the postqreslq

### PR DESCRIPTION
**Pull request by VSemenovykh**

### CODE REVIEW

File: src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        jdbcTemplate.execute(createAuto);
+
+        List<Brand> brandList = new ArrayList<>();
+        brandList.add(new Brand(1L, "BMW", "M8", "2015"));
+        brandList.add(new Brand(2L, "BMW", "M5", "2014"));
+        brandList.add(new Brand(3L, "BMW", "M6", "2016"));
+        brandList.add(new Brand(4L, "Mercedes-Benz", "GLE AMG", "2016"));
+
+        List<Motor> motorList = new ArrayList<>();
+        motorList.add(new Motor(1L, "diseal", 4.0));
```
Comment (AndrewSmrnv) [inline src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java:75]: Maybe "diesel" ?)

Comment (VSemenovykh) [reply src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java:75]: yes, i did  typo)

File: src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        List<Brand> brandList = listTable.get("Brand");
+        List<Motor> motorList = listTable.get("Motor");
+        List<Auto> autoList = listTable.get("Auto");
+
+        for(Brand brand: brandList){
+            brandRepository.save(brand);
+        }
+
+        for(Motor motor: motorList){
+            motorRepository.save(motor);
```
Comment (AndrewSmrnv) [inline src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java:103]: Let's use motorRepository.saveAll()

File: src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        }
+
+        for(ru.ncedu.entity.Auto auto: autoList){
+            autorepository.save(auto);
```
Comment (AndrewSmrnv) [inline src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java:107]: Let's use autorepository.saveAll()

File: src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    }
+
+    public void uploadBaseData(){
+        Map<String, List> listTable =  createTables();
+        List<Brand> brandList = listTable.get("Brand");
+        List<Motor> motorList = listTable.get("Motor");
+        List<Auto> autoList = listTable.get("Auto");
+
+        for(Brand brand: brandList){
+            brandRepository.save(brand);
```
Comment (AndrewSmrnv) [inline src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java:99]: Let's use brandRepository.saveAll()

Comment (VSemenovykh) [reply src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java]: corrected to saveAll

File: src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java (hunk 3 above)
Comment (VSemenovykh) [reply src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java]: corrected to saveAll

File: src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java (hunk 2 above)
Comment (VSemenovykh) [reply src/main/java/ru/ncedu/service/LoadBaseDataServiceImp.java]: corrected to saveAll

**Pull request merged**